<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QLabel" name="koutu_img">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>30</y>
         <width>391</width>
         <height>731</height>
        </rect>
       </property>
       <property name="text">
        <string>img</string>
       </property>
      </widget>
      <widget class="QLabel" name="img_orign">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>751</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>test</string>
       </property>
      </widget>
      <widget class="QComboBox" name="choice_push">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>50</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>推杆1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>推杆2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>推杆3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>推杆4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>推杆5</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="applay">
       <property name="geometry">
        <rect>
         <x>1270</x>
         <y>40</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txt_hsv">
       <property name="geometry">
        <rect>
         <x>1080</x>
         <y>430</y>
         <width>321</width>
         <height>101</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_reset">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>130</y>
         <width>200</width>
         <height>60</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_start">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>30</y>
         <width>200</width>
         <height>60</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_mode">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>230</y>
         <width>200</width>
         <height>60</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <item>
        <property name="text">
         <string>颜色</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>白浅深</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>形状</string>
        </property>
       </item>
      </widget>
      <widget class="QSlider" name="Hmin_slider">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>150</y>
         <width>16</width>
         <height>160</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Hmin</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="Smin_slider">
       <property name="geometry">
        <rect>
         <x>1230</x>
         <y>150</y>
         <width>16</width>
         <height>160</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Smin</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="Vmin_slider">
       <property name="geometry">
        <rect>
         <x>1320</x>
         <y>150</y>
         <width>16</width>
         <height>160</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Vmin</string>
       </property>
       <property name="maximum">
        <number>96</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="txt_hsv_2">
       <property name="geometry">
        <rect>
         <x>1080</x>
         <y>320</y>
         <width>321</width>
         <height>101</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QSlider" name="Hmax_slider">
       <property name="geometry">
        <rect>
         <x>1170</x>
         <y>150</y>
         <width>16</width>
         <height>160</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Hmax</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="Smax_slider">
       <property name="geometry">
        <rect>
         <x>1260</x>
         <y>150</y>
         <width>16</width>
         <height>160</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Smax</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="Vmax_slider">
       <property name="geometry">
        <rect>
         <x>1350</x>
         <y>150</y>
         <width>16</width>
         <height>160</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Vmax</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_output">
           <property name="title">
            <string>GroupBox</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_input">
           <property name="title">
            <string>GroupBox</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Tab3</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" rowspan="3" colspan="4">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>700</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="btn_load">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>60</height>
                </size>
               </property>
               <property name="text">
                <string>加载设置</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_add">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>60</height>
                </size>
               </property>
               <property name="text">
                <string>添加一行</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_apply">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>60</height>
                </size>
               </property>
               <property name="text">
                <string>应用</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>230</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>波特率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>串口选择</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="Light">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>60</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>60</height>
                </size>
               </property>
               <property name="text">
                <string>连接</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>60</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>60</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>60</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>60</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>COM3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM8</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>360</y>
         <width>277</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>页</string>
      </attribute>
      <widget class="QPushButton" name="btn1">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>75</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>bbb</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txt_trg_state">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>491</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
